lib/octicons_react/src/index.js

```diff
@@ -26,6 +26,7 @@ export default function Octicon(props) {
     'aria-label': ariaLabel,
     className,
     height,
+    fill: fill || 'currentColor',
```

Comment:
Could we give `fill` a default value with `defaultProps` instead of using the `||` operator?

Reply:
Done